scripts/gen_oscal.py

```diff
@@ -78,7 +78,14 @@ def generate_model(full_name, out_full_name):
 
 
 def generate_models():
-    """Generate all models including 3rd party."""
+    """
+    Generate all models including 3rd party.
+
+    IMPORTANT NOTE: This script will leave the temporary classes in oscal/tmp for reference, but they should be deleted
+    when no longer needed because some of the operations in the build will seek all .py files, and will try to act
+    on these temporary files - giving an error.  So be sure to delete the oscal/tmp directory when done generating
+    new classes.
+    """
```

Comment:
That sounds like a good issue to address. Shall we create an issue to remove tmp folder automatically?

Reply:
I think it's important for new people exposed to the script to have the tmp files retained so they can see them and understand the subsequent processing.  So I don't know of a way to remove them automatically.  The generation of the classes is inherently hands-on and has manual elements because we have no idea what future releases of OSCAL will look like.